stylelint.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   extends: 'stylelint-config-standard',
   ignoreFiles: '**/_normalize.scss',
+  rules: {
+    'at-rule-no-unknown': null,
```

Comment:
Weirdly, the linter doesn't recognise `@mixin`, even when told it's dealing with Sass.